demo/src/examples/1 User Interaction.svelte

```diff
@@ -40,7 +41,7 @@
   <span slot="title">{meta.name}</span>
 
   <label class="flex items-center gap-3">
-    <input bind:checked={interactive} type="checkbox" disabled={!directions} />
+    <input type="checkbox" bind:checked={interactive} disabled={!directions} />
```

Comment:
Are there linter settings which can spare use from decisions like this one?

Reply:
Hmm... Not sure. There probably are some settings to sort properties in the alphabetical order, but I'm guided by other priorities.